data/reusables/actions/environment-variables-are-fixed.md

```diff
@@ -0,0 +1,4 @@
+
+It is not possible nor advisable to re-set any of the default environment variables (especially those named `GITHUB_*` and `RUNNER_*`) listed in [default environment variables](/actions/learn-github-actions/environment-variables#default-environment-variables) using `GITHUB_ENV`.
+
+At the time of this note, the `CI` variable could be overwritten, but that is not guaranteed.
```

Comment:
```suggestion
It is not possible to overwrite the value of the default environment variables named `GITHUB_*` and `RUNNER_*`. Currently it is possible to overwrite the value of the `CI` variable. However, it's not guaranteed that this will always be possible. 
```